tests/metrics/classification/test_f_beta.py

```diff
@@ -0,0 +1,129 @@
+
+    return fbeta_score(y_true=sk_target, y_pred=sk_preds, average='binary', beta=beta)
+
+
+def _sk_fbeta_multilabel_prob(preds, target, average='micro', beta=1.):
+    sk_preds = (preds.view(-1, NUM_CLASSES).numpy() >= THRESHOLD).astype(np.uint8)
+    sk_target = target.view(-1, NUM_CLASSES).numpy()
+
+    return fbeta_score(y_true=sk_target, y_pred=sk_preds, average=average, beta=beta)
+
+
+def _sk_fbeta_multilabel(preds, target, average='micro', beta=1.):
+    sk_preds = preds.view(-1, NUM_CLASSES).numpy()
+    sk_target = target.view(-1, NUM_CLASSES).numpy()
+
+    return fbeta_score(y_true=sk_target, y_pred=sk_preds, average=average, beta=beta)
+
+
+def _sk_fbeta_multiclass_prob(preds, target, average='micro', beta=1.):
+    sk_preds = torch.argmax(preds, dim=len(preds.shape) - 1).view(-1).numpy()
+    sk_target = target.view(-1).numpy()
+
+    return fbeta_score(y_true=sk_target, y_pred=sk_preds, average=average, beta=beta)
+
+
+def _sk_fbeta_multiclass(preds, target, average='micro', beta=1.):
+    sk_preds = preds.view(-1).numpy()
+    sk_target = target.view(-1).numpy()
+
+    return fbeta_score(y_true=sk_target, y_pred=sk_preds, average=average, beta=beta)
+
+
+def _sk_fbeta_multidim_multiclass_prob(preds, target, average='micro', beta=1.):
+    sk_preds = torch.argmax(preds, dim=len(preds.shape) - 2).view(-1).numpy()
+    sk_target = target.view(-1).numpy()
+
+    return fbeta_score(y_true=sk_target, y_pred=sk_preds, average=average, beta=beta)
+
+
+@pytest.mark.parametrize("ddp", [True, False])
+@pytest.mark.parametrize("dist_sync_on_step", [True, False])
+@pytest.mark.parametrize("average", ['micro', 'macro'])
+@pytest.mark.parametrize("preds, target, sk_metric, num_classes, multilabel", [
+    (_testing_binary_prob_inputs.preds, _testing_binary_prob_inputs.target, _sk_fbeta_binary_prob, 1, False),
+    (_testing_binary_inputs.preds, _testing_binary_inputs.target, _sk_fbeta_binary, 1, False),
+    (_testing_multilabel_prob_inputs.preds, _testing_multilabel_prob_inputs.target, _sk_fbeta_multilabel_prob, NUM_CLASSES, True),
+    (_testing_multilabel_inputs.preds, _testing_multilabel_inputs.target, _sk_fbeta_multilabel, NUM_CLASSES, True),
+    (_testing_multiclass_prob_inputs.preds, _testing_multiclass_prob_inputs.target, _sk_fbeta_multiclass_prob, NUM_CLASSES, False),
+    (_testing_multiclass_inputs.preds, _testing_multiclass_inputs.target, _sk_fbeta_multiclass, NUM_CLASSES, False),
+    (
+            _testing_multidim_multiclass_prob_inputs.preds,
+            _testing_multidim_multiclass_prob_inputs.target,
+            _sk_fbeta_multidim_multiclass_prob,
+            NUM_CLASSES,
+            False
+    ),
+    (
+            _testing_multidim_multiclass_inputs.preds,
+            _testing_multidim_multiclass_inputs.target,
+            _sk_fbeta_multiclass,
+            NUM_CLASSES,
+            False
+    )
+])
+@pytest.mark.parametrize("metric_class, beta", [(FBeta, 0.5), (FBeta, 1.)])
+def test_fbeta(
```

Comment:
I cannot find where is some comparison or our and ask metric, can you please make it more obvious...

Reply:
test_f_beta.py contains fbeta_score metric in return values for sk_metric function